source/Calamari.Tests/KubernetesFixtures/Terraform/Clusters/EKS/eks.kubernetes.tf

```diff
@@ -13,8 +13,8 @@ provider "kubernetes" {
   token                  = data.aws_eks_cluster_auth.default.token
 }
 
-resource "local_file" "kubeconfig" {
-  sensitive_content = templatefile("${path.module}/kubeconfig.tpl", {
+resource "local_sensitive_file" "kubeconfig" {

```

Comment:
I was getting a test failure because `sensitive_content` is not supported in the `local_file` resource anymore. I've updated it to use the new `local_sensitive_file` instead.